pkg/manager/manager.go

```diff
@@ -133,7 +136,7 @@ func (m *Manager) startNorthboundServer(
 
 	s.AddService(logging.Service{})
 
-	adminService := admin.NewService(networkChangesStore, networkSnapshotStore, deviceSnapshotStore, pluginRegistry)
+	adminService := admin.NewService(transactionsStore, configurationsStore, pluginRegistry, networkChangesStore, networkSnapshotStore, deviceSnapshotStore)
```

Comment:
I think we should remove networkChangeStore, snaphort store, deviceStore from the arguments and clean up admin service.

Reply:
Cleaned up as much as possible at this point in time. Left notes in other places.